cmd/fluvia/main.go

```diff
@@ -51,5 +51,10 @@ func main() {
 		ingressIfName = c.Ipfix.IngressInterface
 	}
 
-	client.New(ingressIfName, raddr)
+	interval := c.Ipfix.Interval
```

Comment:
It would be nice if the readme could understand how to use this option.

Reply:
I wrote how to configure interval in getting-started.md